pages.sr/common/git.md

```diff
@@ -0,0 +1,29 @@
+# git
+
+> Distribuirani sistem kontrole verzija.
+> Neke podkomande kao što je `git commit` imaju svoj primer u dokumentaciji.
+> More information: <https://git-scm.com/>.
```

Comment:
translate as well to: "Više informacija na"

Reply:
Thanks for noticing it, I fixed it in both places.